src/views/About/index.js

```diff
@@ -0,0 +1,151 @@
+            <h4>
+              <Typography type="subheading" gutterBottom>
+                Incubation Team
+              </Typography>
+            </h4>
+            <p>
+              <Typography type="body2" gutterBottom>
+              Ezeanyinabia ‘Eze’ Anyanwu ’17, Adam Bartholomew ’17, and James Kempf ‘17
+                <br /> Dr. Russ Tuck, Summer Practicum in Computer Science program (2016)
+              </Typography>
+            </p>
+          </div>
+          <div>
+            <h4>
+              <Typography type="subheading" gutterBottom>
+              Product Deployment and Support
+              </Typography>
+            </h4>
+            <p>
+              <Typography type="body2" gutterBottom>
+              Emmanuel Roussel ‘17 and Stephanie Powers &apos;17
+                <br /> Chris Hansen; Jason Whitehouse ’99, Information Systems Group
+              </Typography>
+            </p>
+          </div>
+          <div>
+            <h4>
+              <Typography type="subheading" gutterBottom>
+              Software Development Team, 2016/2017
+              </Typography>
+            </h4>
+            <p>
+              <Typography type="body2" gutterBottom>
+              Ross and Chris Qiao &apos;17
+                <br />Dr. Russ Tuck, Dr. Jonathan Senning, Summer Practicum in Computer Science
+                program
+              (2017)
+              </Typography>
+            </p>
+          </div>
+          <div>
+            <h4>
+              <Typography type="subheading" gutterBottom>
+              Software Development Team, 2017/2018
+              </Typography>
+            </h4>
+            <p>
+              <Typography type="body2" gutterBottom>
+              Addison Abbot &apos;20, Matt Felgate &apos;18, and Rebekah Stauffer &apos;19
+              </Typography>
+            </p>
+          </div>
+          <div>
+            <h4>
+              <Typography type="subheading" gutterBottom>
+              Beneficient Philanthropes
+              </Typography>
+            </h4>
+            <p>
+              <Typography type="body2" gutterBottom>
+              To the visionary benefactors who support the Summer Practicum in Computer Science
+              program at Gordon College, thank you!
+                <br /> And to Dr. D. Michael Lindsay, President; Jennifer Jukanovich ’94, Vice
+              President for Student Life, thank you for
+              the opportunity to apply our learning in a creative, practical way!
+                <br /> And to our mentors and coaches in development and deployment, thank you for
+               your guidance and encouragement to try new things.
+              </Typography>
+            </p>
+          </div>
+          <div>
+            <h5>
+              <Typography type="caption" gutterBottom>
+              Found a bug? Report to
+                <a href="mailto:cts@gordon.edu?Subject=Gordon 360 Bug" target="_top"> CTS</a>
```

Comment:
Instead of just "CTS" being a link, I think "Report to CTS" should be a link. This will make it more obvious to users that the text is clickable.

Also, I don't think we need `target="_top"` here - it doesn't have any effect, as far as I can tell.